src/app/account-management/general-information-form/general-information-form.component.html

```diff
@@ -43,7 +43,7 @@
 
         <div class="input-group mb-3 col">
             <label>Facility Area</label>
-            <input type="text" class="form-control" formControlName="size" (input)="saveChanges()" onfocus="this.select();">
+            <input type="text" class="form-control" formControlName="size" (input)="saveChanges()" onfocus="this.select();" minlength="1" maxlength="999999999999999">
```

Comment:
I think you could probably limit this to around 60. At the point its 999999999999999, you wouldn't need to put a max on it.